spec/views/hq/orphans/show_spec.rb

```diff
@@ -11,7 +11,9 @@
       expect(rendered).to match @orphan.name
```

Comment:
When we added these `CGI::escape_html` I remember that it wouldn't work without it. Names like `O'Conner` would fail withtout CGI::escape.
Now it looks like it's the other way around. Maybe this feature was implemented by default in rails 4.2?